kuma/users/management/commands/export_to_sendinblue.py

```diff
@@ -0,0 +1,89 @@
+import csv
+import json
+from io import StringIO
+
+import requests
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+URL = "https://api.sendinblue.com/v3/contacts/import"
+
+
+def _dictfetchall(cursor):
+    "Returns all rows from a cursor as a dict. source: https://stackoverflow.com/a/14294314"
+    desc = cursor.description
+    return [dict(zip([col[0] for col in desc], row)) for row in cursor.fetchall()]
+
+
+class Command(BaseCommand):
+    help = "Exports newsletter subscribed users to sendinblue"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--api-key")
+        parser.add_argument("--paying-users-list-id", type=int)
+        parser.add_argument("--non-paying-users-list-id", type=int)
+
+    def handle(self, **options):
+        cursor = connection.cursor()
+        cursor.execute(
+            """
+            SELECT
+                email,
+                first_name,
+                last_name,
+                EXISTS(
+                    SELECT 1
+                    FROM users_usersubscription
+                    WHERE user_id = auth_user.id AND canceled IS NULL
+                ) AS is_paying
+            FROM auth_user
+            WHERE email <> '' AND is_newsletter_subscribed
+        """
+        )
```

Comment:
An extremely simple solution would be this:
```python
active_subscriber_user_ids = set(
    UserSubscription.objects.filter(canceled__isnull=True).values_list('user_id', flat=True)
)
users = User.objects.filter(is_newsletter_subscribed=True).exclude(email='')
for user in users.only('email', 'first_name', 'last_name'):
    is_paying = user.id in active_subscriber_user_ids
    print(user.email, is_paying)
```

Since the `active_subscriber_user_ids` set is unlikely to be in the tens of thousands, it's fine to hold it all in memory. 
The rest is about as fast and memory-efficient as can possibly be.

Reply:
neat, thanks for django-fying it, that's definitely better than the things I came up with. You and @escattone  and most django-istas would find that preferable over the raw SQL, right?

Reply:
Any day!
Raw SQL should be avoided at all costs since the ORM makes us agnostic. However, easier said than done. 

Also, your command is run once and then cleaned up. But when you use `connection.cursor()` you should use a context manager so it closes itself.

```python
with connection.cursor() as cursor:
    cursor.execute(sql_string)
```